<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" verbose="5" name="Failed suite [Suite]">
  <test thread-count="5" verbose="5" name="Test(failed)">
    <classes>
      <class name="tests.selenium.TC002_CreateIncidentAndGetFromRest">
        <methods>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="afterSuite"/>
          <include name="createIncident" invocation-numbers="0"/>
          <include name="afterMethod"/>
          <include name="beforeSuite"/>
          <include name="setValues"/>
        </methods>
      </class> <!-- tests.selenium.TC002_CreateIncidentAndGetFromRest -->
      <class name="tests.selenium.TC003_CreateIncidentUsingRestAndVerifyUsingSelenium">
        <methods>
          <include name="createIncident" invocation-numbers="0"/>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="afterSuite"/>
          <include name="setValues"/>
          <include name="afterMethod"/>
          <include name="beforeSuite"/>
        </methods>
      </class> <!-- tests.selenium.TC003_CreateIncidentUsingRestAndVerifyUsingSelenium -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
